src/Service/FixtureService.php

```diff
@@ -28,49 +33,85 @@
  */
 class FixtureService
 {
+    /**
+     * @var Loader
+     */
     private $loader;
+
+    /**
+     * @var bool
+     */
     private $autoload;
+
+    /**
+     * @var array
+     */
     private $fixtures;
+
+    /**
+     * @var array
+     */
     private $directories;
-    private $migrations;
+
+    /**
+     * @var Kernel
+     */
     private $kernel;
+
+    /**
+     * @var EntityManagerInterface
+     */
     private $entityManager;
+
+    /**
+     * @var PlatformListener
+     */
     private $listener;
-    private $useBackup;
+
+    /**
+     * @var null|BackupService
+     */
     private $backupService;
 
     /**
-     * @var \Doctrine\Common\DataFixtures\ProxyReferenceRepository
+     * @var null|ProxyReferenceRepository
      */
     private $referenceRepository;
 
     /**
-     * Constructor.
+     * FixtureService constructor.
      *
-     * @param \Symfony\Component\DependencyInjection\ContainerInterface $container Service container
-     * @param \Symfony\Component\HttpKernel\Kernel                      $kernel    Application kernel
+     * @param Kernel $kernel
+     * @param bool   $autoload
+     * @param array  $fixtures
+     * @param array  $directories
```

Comment:
I think this one and above could be like `string[]`

Reply:
Done